domain/schema/model/sql/0024-relation.sql

```diff
@@ -236,4 +233,16 @@ SELECT
     rs.suspended_reason,
     rs.updated_at
 FROM relation_status AS rs
-JOIN relation_status_type AS rst ON rs.relation_status_type_id = rst.id
+JOIN relation_status_type AS rst ON rs.relation_status_type_id = rst.id;
+
+CREATE VIEW v_relation_unit_status AS
```

Comment:
Given that the use of this view is quite esoteric - a unit-eye view of all relations its application is in, it's unlikely to be useful outside the single usage in state.

Just put this query directly where the view is being selected from.